fiftyone/core/expression_catalog.py

```
@@ -0,0 +1,478 @@
+                min_args=min_args,
+                max_args=max_args,
+                reflected=name in _REFLECTED,
+                typed=name in _KINDS,
+                summary=_summary(attr),
+            )
+        )
+
+    return sorted(specs, key=lambda s: s["name"])
+
+
+def _arity(func, static: bool) -> Tuple[int, Optional[int]]:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)